pages/common/peerflix.md

```diff
@@ -0,0 +1,19 @@
+# peerflix
+
+> Stream torrents to your favorite player.
+
+- Stream a torrent:
+
+`peerflix "{{torrent url|magnet link}}"`
```

Comment:
The quotes are required, right? Would a concrete example here be possible?

Reply:
It works without them, but depending they are used in the official man, i guess it depends a lot on what type of characters are used in the magnet link.

I was not sure about the concrete example. I checked out examples like 'screenfetch' as inspiration.

Reply:
Ah, ok.  Ping @agnivade /  @waldyrious - what do you think?

Reply:
Replied (kind of, since the source has changed) [here](https://github.com/tldr-pages/tldr/pull/1457#discussion_r135742461).